public/app/plugins/datasource/loki/components/LokiQueryField.tsx

```diff
@@ -92,7 +92,7 @@ export class LokiQueryField extends React.PureComponent<LokiQueryFieldProps, Lok
     ];
   }
 
-  async componentDidUpdate() {
+  async componentDidMount() {
```

Comment:
No idea why this was ever on update. Should have been on mount from the beginning. Blame pointed towards me 🌮

Reply:
Wondering if this is going to fix https://github.com/grafana/grafana/issues/32565

Reply:
Yup, high chance!